CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+- Optimises memory usage in IF calculation and returns infor about hessian
```

Comment:
As the return type of a public function of the package is modified in a non backward compatible way, please mark this as a breaking change